src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartBoundaryLevelsConfiguration.test.tsx

```diff
@@ -144,62 +150,79 @@ describe('ChartBoundaryLevelsConfiguration', () => {
     expect(row1Cells[0]).toHaveTextContent(
       'Number of authorised absence sessions (Ethnicity Major Chinese, State-funded primary, All locations, 2014/15)',
     );
-    expect(row1Cells[1]).toHaveTextContent('Use default');
+    expect(within(row1Cells[1]).getByRole('combobox')).toHaveValue('');
+
+    const row2Cells = within(rows[2]).getAllByRole('cell');
+    expect(row2Cells[0]).toHaveTextContent(
+      'Number of authorised absence sessions (Ethnicity Major Chinese, State-funded primary, All locations, 2015/16)',
+    );
+    expect(within(row2Cells[1]).getByRole('combobox')).toHaveValue('');
+
+    rows.forEach((row, rowIndex) => {
+      if (rowIndex === 0) return;
+      const cells = within(row).getAllByRole('cell');
+      const boundaryLevelOptions = within(cells[1]).getAllByRole('option');
+      boundaryLevelOptions.forEach((option, optionIndex) => {
+        if (optionIndex === 0) {
+          expect(option).toHaveTextContent('Use default');
+          expect(option).toHaveValue('');
+        } else {
+          const { id, label } = testMeta.boundaryLevels[optionIndex - 1];
+          expect(option).toHaveTextContent(label);
+          expect(option).toHaveValue(String(id));
+        }
+      });
+    });
```

Comment:
Wouldn't advise using for loops with conditional logic like this in test cases. We want things to be as clear as possible even at the expense of a bit of copy and paste.

Keep things simple and just check things using indices.

Reply:
I've simplified the code to be more obvious, stating that the options should match up with the supplied boundary levels in the data - I've removed the conditional in the loop. I'll take this forward, keeping tests simple and obvious, avoiding loops with the idea saving lines isn't worth the complexity in tests.